README.md

```diff
@@ -34,14 +34,14 @@ GUARD Tool
 Usage: guard [OPTIONS]
 
 Options:
-  -h,--help              Use the below listed functions.
-                         Warning: Don't try guard on non guardable units
-                         (sys, perv)
-  -c,--create TEXT       Create GUARD record, expects physical path as input.
-  -d,--delete TEXT       Delete GUARD record, expects physical path as input.
-  -l,--list              Listing of GUARDed resources.
-  -r,--clearall          Clears GUARD states for all resources.
-  -v,--version           Version of GUARD tool.
+  -h,--help                Guard CLI tool options
```

Comment:
Commit message : Add  details "why this changes required?" 
based on your initial commit message comments  -r is not changing to -I , it is juts changing -clear-all option, looks like you are removing -r option here , contradicyting the initail commmit response. ,update accordingly , refder precious  comments .  Stopping the review now, will continue receview , once you clarified the direction .

Reply:
Done.